apps/server/prisma/schema.prisma

```diff
@@ -10,40 +10,46 @@ datasource db {
 }
 
 model Todo {
-  id       String    @id @default(auto()) @map("_id") @db.ObjectId
-  todo  String
+  id   String @id @default(auto()) @map("_id") @db.ObjectId
+  todo String
 }
 
 model User {
-  id       String    @id @default(auto()) @map("_id") @db.ObjectId
+  id         String   @id @default(auto()) @map("_id") @db.ObjectId
   name       String
   image      String?
   register   Register @relation(fields: [registerId], references: [id])
-  registerId String @unique
+  registerId String   @unique
 }
 
 model Register {
-  id       String    @id @default(auto()) @map("_id") @db.ObjectId
-  email        String   @unique
-  phone        String   @unique
-  password     String
-  role         String
+  id          String @id @default(auto()) @map("_id") @db.ObjectId
+  email       String @unique
+  phone       String @unique
+  password    String
+  role        String
   accessToken String
-  user         User?
+  user        User?
 }
-model FeedahelpNewsletter{
-    id       String    @id @default(auto()) @map("_id") @db.ObjectId
-    userId String?
-    email String @unique
-    createAt DateTime @default(now())
+
+model FeedahelpNewsletter {
+  id       String   @id @default(auto()) @map("_id") @db.ObjectId
+  userId   String?
+  email    String   @unique
+  createAt DateTime @default(now())
+}
+
+model FahCounter {
+  id                 String @id @default(auto()) @map("_id") @db.ObjectId
+  sectionName        String
+  sectionDescription String
+  imageIconUrl       String
+  imageAlt           String
+  value              String
+  monthlyIncrease    String
 }
 
-model FahCounter{
-    id       String    @id @default(auto()) @map("_id") @db.ObjectId
-    sectionName String
-    sectionDescription String
-    imageIconUrl String
-    imageAlt String
-    value String
-    monthlyIncrease String
-}
\ No newline at end of file
+model Ingredient {
```

Comment:
Plural: `Ingredients`

Reply:
I think, the Singular Model name convention is more used widely. I have checked out some references which suggest singular model name in most cases. 

https://stackoverflow.com/questions/338156/table-naming-dilemma-singular-vs-plural-names

One logic seems apealing to me which states: 

> How about this as a simple example:
> 
> SELECT Ingredient.name FROM Ingredient WHERE Ingredient.name > "def"
> vs.
> SELECT Ingredients.Name FROM Ingredients WHERE Ingredients.name > "def"
> The SQL in the latter is stranger sounding than the former.

However, our previous model names seem to following singular names like User, Todo, Question, Game etc.
After all, this is my opinion and open to dicussion.

Reply:
I agree if its SQL statement.

we will doing in javaScripts when we will have MODELS and its will be name `Ingredients` with plural, which in return will tell us that we have array of ingredients and calling it will be very dangerous because you will calling 3000+ all at once. we will have to `chunk` it or something

make it plural for now. if its problem we can always change it

Reply:
Okay bhai.